recipe/meta.yaml

```diff
@@ -7,8 +7,8 @@ source:
     git_tag: ESMF_7_0_0
 
 build:
-    number: 1
-    skip: True  # [win or py3k]
+    number: 2
+    skip: True  # [win]
```

Comment:
And no need to bump the build number in this case. (There are no Python 3 packages at the moment and Python 2 build number 1 will be the same as 2.)